docs/sdkx-react-native/sdk-configuration.mdx

```diff
@@ -0,0 +1,306 @@
+}
+let appId = Platform.select({
+  android: <Android_Platform_Id>,
+  ios: <iOS_Platform_Id>,
+});
+
+if (appId != null) {
+  install(appId, <domain>, installConfig);
+} else {
+  //Display your error here.
+}
+```
+
+## Full privacy {#fullprivacy}
+
+In the `config` dictionary of the `showConversation` API at the time of calling this API, setting the `fullPrivacy` option to `true` ensures COPPA compliance by:
+
+<div className="compact ">
+
+|        |               |
+| ------ | --------------|
+| Flag   | `fullPrivacy` |
+| values | true/ false   |
+
+</div>
+
+1. Disabling user-initiated screenshots (users will not be able to attach files, including images, using SDK).
+2. Make sure that Personally Identifiable Information (PII) such as name and email are not collected by SDK (using Identity Bot and/or the helpshiftConfig object). This means that if you set `userName` and `userEmail`, with `fullPrivacy` set to `true`, Helpshift will not use the `userName` and `userEmail` values.
+
+Moreover, in scenarios where the user attaches objectionable content, it becomes a huge COPPA compliance concern. This option helps to solve this problem.
+This config is common for both iOS and Android.
+```
+let installConfig = {
+  'fullPrivacy': true,
+}
+let appId = Platform.select({
+  android: <Android_Platform_Id>,
+  ios: <iOS_Platform_Id>,
+});
+
+if (appId != null) {
+  install(appId, <domain>, installConfig);
+} else {
+  //Display your error here.
+}
+```
+
+## Enable Contact Us {#enableContactUs}
+
+Controls the visibility of the Helpshift Contact Us button when a user is viewing FAQs. You can customize this option to make it easier or more difficult to contact support. If specified, this configuration takes precedence over the value of **Enable Contact Us** set on the admin dashboard.
+
+<div className="compact ">
+
+|         |                                                                                                                                  |
+| ------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| Key     | <code>enableContactUs</code>                                                                                                   |
+| values  | <code>"ALWAYS"</code> / <code>"AFTER_VIEWING_FAQS"</code> / <code>"AFTER_MARKING_ANSWER_UNHELPFUL"</code> / <code>"NEVER"</code> |
+| default | none                                                                                                                             |
+
+</div>
+
+```
+showFAQsWithConfig({
+  faq: <FAQ_SECTION_ID>,
+  enableContactUs: `AFTER_VIEWING_FAQS`
+});
+```
+
+#### Best Practices
+
+- Provide tier-based support by setting enableContactUs to `ALWAYS` for paid users and `AFTER_VIEWING_FAQS` for unpaid ones.
+- Provide country-based support by setting enableContactUs to `ALWAYS` for local users and `AFTER_VIEWING_FAQS` for foreign ones.
+
+## Android Specific Configurations
```

Comment:
Need code snippets for all the configs here.

Reply:
@vikashelpshift  please update it for Android.